### ISSUE TITLE
Commands defined via ALIASES tag have an extra space after value

### ISSUE THREAD

**Issue by rolegic** [doxygen/doxygen#10912]
**Describe the bug**
An extra space is added after each command value.

**Screenshots**
N/A

**To Reproduce**
- Input:
  ```c
  /*!
  Install path: "@drive/@path/@bin"
  */
  ```
- Doxyfile:
  ```
  ALIASES  = "drive=C:" \
             "path =path/to" \
             "bin  =program.exe"
  ```
**Expected behavior**
→ Until doxygen 1.10
```
Install path: "C:/path/to/program.exe"
```

**Unexpected behavior**
→ Regression with doxygen 1.11.0
```
Install path: "C: /path/to /program.exe "
```

**Version**
- Windows 11, 64 bit (binary from doxygen website)
- Ubuntu 22.04.4 LTS (binary from doxygen website)

**Stack trace**
N/A

**Additional context**
N/A

**Comment by albert-github** [comment]
A bisection gave (1099c80cb6d94718e8235a8a916f8c8b99b6fe23):
```
1099c80cb6d94718e8235a8a916f8c8b99b6fe23 is the first bad commit
Date:   Mon Feb 5 20:57:22 2024 +0100

    issue #10606 include command via ALIASES does not work anymore

 doc_internal/commands_internal.md | 11 +++++++++++
 src/commentcnv.l                  | 38 +++++++++++++++++++++++++++++++++++---
 2 files changed, 46 insertions(+), 3 deletions(-)
```
Probably there is also some interaction with (3f4d6a4d0cf713540b6a86c19014c8048e7beed5):
```
Commit: 3f4d6a4d0cf713540b6a86c19014c8048e7beed5 [3f4d6a4]
Date: Monday, February 5, 2024 8:52:25 PM

Fix issue with space after alias name
```

**Comment by albert-github** [comment]
A first quick thought was: Can the space in `result += " \\ialias{";` in the function `static void replaceAliases(yyscan_t yyscanner,std::string_view s,bool replaceComment)` (commentcnv,l) be removed but this gives problems with the original example in the original issue #10606 so some "intelligent" handling of the `ialias` command is required.

I've just pushed a proposed patch, pull request #10914 (also the results of #10606 look still OK)

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by rolegic** [comment]
Work as expected!

Thanks for your great job and amazing tool 🥇
